### ISSUE TITLE
Is there a variable version of this font available? 

### ISSUE THREAD

**Issue by o-t-w** [latofonts/lato-source#35]

**Comment by applecuckoo** [comment]
Yes there is, I've built them and stored it in a repository named applecuckoo/lato-builds. Here's the link to the two files:

[Regular](https://github.com/applecuckoo/lato-builds/tree/main/Lato3Upr2M/vf-3master)
[Italic](https://github.com/applecuckoo/lato-builds/tree/main/Lato3Ita2M/vf-3master)

**Comment by tettoffensive** [comment]
@applecuckoo Are there webfont versions of the variable font available?

**Comment by applecuckoo** [comment]
@tettoffensive Hi! There weren't but I've just generated some with .woff2 compression that you should be able to use. Those have been pushed out to the repo, they should be in the same locations that I linked above.
